imaging/imageRendering.ts

```diff
@@ -790,6 +804,22 @@ export const renderImage = function (
             `updating cornerstone viewport with custom contrast values: ${renderOptions.voi.windowWidth}, ${renderOptions.voi.windowCenter}`
           );
         }
+        // Update the viewport VOI values with the default values from the series
+        // if the VOI is not defined in the renderOptions.
+        // Note: Not all images have the same VOI values, so this update is needed
+        // even when the uniqueUID hasn't changed but the slice has changed.
+        else {
+          viewport.voi!.windowWidth =
+            data.default?.voi?.windowWidth || image.windowWidth;
+          viewport.voi!.windowCenter =
+            data.default?.voi?.windowCenter || image.windowCenter;
+          logger.debug(
+            "updating cornerstone viewport with default voi values: ",
+            viewport.voi!.windowWidth,
+            viewport.voi!.windowCenter
+          );
+        }
```

Comment:
quel pezzo qua non va bene.
Se visualizzo ad esempio una immagine con tanti frame e cambio ww/wl non funziona piu nulla perchè ogni volta forzi un contrasto diverso. Non ho capito cosa volessi fare ma cosi in ogni caso è "sbagliato" xD
Se usi esempio multiframe attivando tool del contrasto te ne accorgi subito ad esempio